bitnami/etcd/templates/statefulset.yaml

```
@@ -20,12 +20,12 @@ spec:
   template:
     metadata:
       labels: {{- include "etcd.labels" . | nindent 8 }}
-      {{- if or .Values.podAnnotations .Values.metrics.enabled }}
+      {{- if or .Values.podAnnotations (and .Values.metrics.enabled (not .Values.metrics.serviceMonitor.enabled)) }}
```

I think this is also necessary to be consistent with the other two conditions below:

```suggestion
      {{- if or .Values.podAnnotations (and .Values.metrics.enabled .Values.metrics.podAnnotations (not .Values.metrics.serviceMonitor.enabled)) }}
```